client-app/pages/product.vue

```diff
@@ -8,8 +8,18 @@
 
       <h1 class="text-2xl font-bold uppercase md:text-4xl">{{ product.name }}</h1>
 
-      <div v-if="!product.hasVariations && !isMobile" class="mt-1 text-sm">
-        {{ $t("pages.product.sku_label") }} <span class="font-extrabold">{{ product.code }}</span>
+      <div v-if="!product.hasVariations" class="mt-1 flex items-center text-sm">
```

Comment:
As I remember we wanted to create shared component, ain't we?

Reply:
added